### PR TITLE
Fix for JKS to PKCS12 change

### PR DESCRIPTION
**Pull request by prb112**
Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

File: fhir-operation-bulkdata/src/test/java/com/ibm/fhir/operation/bulkdata/client/BulkDataClientMain.java [hunk 1]
```diff
@@ -20,7 +20,8 @@ public static void main(String... args) throws Exception {
         properties.put(BulkDataConfigUtil.BATCH_USER, "fhiradmin");
         properties.put(BulkDataConfigUtil.BATCH_USER_PASS, "change-password");
                 
-        properties.put(BulkDataConfigUtil.BATCH_TRUSTSTORE, "/Users/paulbastide/git/wffh/wlp/wlp/usr/servers/defaultServer/resources/security/fhirTruststore.jks");
+        properties.put(BulkDataConfigUtil.BATCH_TRUSTSTORE,
+                "/Users/paulbastide/git/wffh/wlp/wlp/usr/servers/defaultServer/resources/security/fhirTruststore.p12");
```
Comment (lmsurpre) [inline fhir-operation-bulkdata/src/test/java/com/ibm/fhir/operation/bulkdata/client/BulkDataClientMain.java:24]: can it be a relative path?
if not, then maybe add a placeholder for git and a comment to say you should change it to the path on your system?

Comment (prb112) [reply fhir-operation-bulkdata/src/test/java/com/ibm/fhir/operation/bulkdata/client/BulkDataClientMain.java:24]: Fixed

File: fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulktools/Main.java [hunk 2]
```diff
@@ -97,7 +97,7 @@
      * Parse the command line arguments
      *   --cosapikey:       the IBM COS API key or S3 access key.
      *   --cossrvinstid:    the IBM COS service instance id or S3 secret key.
-     *   --cosendpinturl:   the IBM COS or S3 End point URL.
+     *   --cosEndpointUrl:   the IBM COS or S3 End point URL.
      *   --coslocation:     the IBM COS or S3 location.
      *   --cosbucketname:   the IBM COS or S3 bucket name to import from.
      *   --coscredentialibm: if use IBM credential(Y/N), default(Y).
```
Comment (lmsurpre) [inline fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulktools/Main.java:103]: the mix of all-lower and camelCase looks odd to me

Comment (prb112) [reply fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulktools/Main.java:103]: I can't speak to it

Comment (prb112) [reply fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulktools/Main.java]: fixed

**Pull request merged**